src/heuristics.jl

```diff
@@ -2,3 +2,78 @@
 
 ## Have a general interface for heuristics 
 ## so that the user can link in custom heuristics.
+"""
+    Boscia Heuristic
+
+Interface for heuristics in Boscia.    
+`h` is the heuristic function receiving as input the tree, the bounded LMO and a point x (the current node solution).
+It returns the heuristic solution (can be nothing, we check for that) and whether feasibility still has to be check.
+`prob` is the probability with which it will be called.        
+"""
+# Would 'Heuristic' also suffice? Or might we run into Identifer conflicts with other packages?
+struct Heuristic
+    h::Function
+    prob::Float64
+    identifer::Symbol
+end
+
+# Create default heuristic. Doesn't do anything and should be called.
+Heuristic() = Heuristic(x -> nothing, -0.1, :default)
+
+"""
+Flip coin.
+"""
+function flip_coin(w=0.5)
+    return rand() ≤ w
+end
+
+"""
+Add a new solution found from the heuristic to the tree.
+"""
+function add_heuristic_solution(tree, x, val, heu::Symbol)
```

Comment:
```suggestion
function add_heuristic_solution(tree, x, val, heuristic_name::Symbol)
```

better use complete variable names